engine/kernel.gyp

```diff
@@ -83,12 +83,16 @@
 							'src/mblcamera.cpp',
 						],
 										
-						# Force the entry point to be included in the output
+						
 						'link_settings':
 						{
 							'ldflags':
 							[
-								'-Wl,--undefined,Java_com_runrev_android_Engine_doCreate'
+								# Force the entry point to be included in the output
+								'-Wl,--undefined,Java_com_runrev_android_Engine_doCreate',
+								
+								# Force inclusion of LCB Invocation Handler native callback
```

Comment:
WE should embelish this comment to explain why - i.e. because the file contains nothing other than the native (JNI) function, which means the object gets discarded by the linker otherwise as nothing in it is statically used.